packages/jellyfish-api-core/src/category/icxorderbook.ts

```diff
@@ -96,6 +96,49 @@ export class ICXOrderBook {
       'bignumber'
     )
   }
+
+  /**
+   * Returns information about offers for passed order
+   *
+   * @param {} options
+   * @param {string}  [options.orderTx] Order txid to list all offers for this order
+   * @return {Promise<Record<string, ICXOfferInfo>>} Object indluding details of offers.
+   */
+  async listOrders (options: { orderTx: string }): Promise<Record<string, ICXOfferInfo>>
```

Comment:
```suggestion
  async listOrders (options: { orderTx: string } & ICXListOrderOptions): Promise<Record<string, ICXOfferInfo>>
```

You need this or else it won't work with other options